lib/webpacker/compiler.rb

```diff
@@ -53,7 +53,7 @@ def record_compilation_digest
     def run_webpack
       logger.info "Compiling…"
 
-      sterr, stdout, status = Open3.capture3(webpack_env, "#{RbConfig.ruby} ./bin/webpack")
+      sterr, stdout, status = Open3.capture3(webpack_env, "bundle exec webpack")
```

Comment:
`./bin/*` binstubs aren't really needed when using `bundle exec`, right? If so, the `check_binstubs` name isn't really accurate. We only care that webpacker the gem is installed.

Reply:
Yeah, I guess the whole`./bin` thing is now irrelevant. Will clean this up in another PR